### ISSUE TITLE
Encoding Control 3

### ISSUE THREAD

**Issue by josejimenezp** [IIC2413/Syllabus-2020-2#459]
Hola, tengo un solo problema con el control 3. En la pregunta 2.2, cuando creo la tabla me quedan las cosas con otro encoding, no sé si esto es necesario arreglarlo (si tiene descuento dejarlo así o no). Adjunto foto.

![image](https://user-images.githubusercontent.com/48298035/101090657-c0719300-3595-11eb-9796-3e3380197a07.png)

Muchas gracias

**Comment by juanreutter** [comment]
No importa, de hecho podrías usar los ids en vez de los nombres y no tendrías problema alguno.

**Comment by rodrigobj99** [comment]
Hola! Entonces para la 2.2 podemos crear en vez de ComunaLimpia(id,nombre) lo mismo pero sin nombre como ComunaLimpia(id), y con ello evitar cambiar la forma de conectarnos a la base de datos? (ya que el problema ocurre por acento en los nombres)

De antemano muchas gracias!

**Comment by juanreutter** [comment]
No, pero si pueden crear la vista EN la base de datos (no en pandas).

**Comment by rodrigobj99** [comment]
Gracias!!
